docs/samples/advanced_config/aggregator.yaml

```diff
@@ -211,3 +211,13 @@ task_cache_ttl_s: 600
 # Defines how many tasks can be cached. This affects how much memory the
 # aggregator might use to store cached tasks. (optional)
 task_cache_capacity: 10000
+
+# Queue aggregate init and continue requests with a LIFO strategy. (optional;
+# experimental, may be removed or changed in minor releases)
+helper_request_queue:
```

Comment:
The `helper_request_queue` name isn't quite right since it does not affect `AggregateShare` requests.

Maybe `helper_aggregation_request_queue`?